x/blob/client/cli/payforblob.go

```diff
@@ -57,6 +75,7 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().String("namespace-version", "", "User can use this to specify the namespace version when they submit a pay for blob.")
```

Comment:
once we extract a const for the flag name, we can use it here

Reply:
Updated